Pipfile

```diff
@@ -24,14 +24,14 @@ djangorestframework-simplejwt = "==5.3.1"
 dry-rest-permissions = "==0.1.10"
 drf-nested-routers = "==0.94.1"
 drf-spectacular = "==0.27.2"
-"fhir.resources" = "==6.5.0"
+"fhir.resources" = "==7.1.0"
```

Comment:
we should move this to the hcx plug itself as its not required by anything else

cc: @vigneshhari